docs/querying/sql.md

```diff
@@ -423,3 +423,17 @@ SELECT count(city) from druid.table where SCALAR_IN_ARRAY(city, ?)
 ```
 
 sample java code using dynamic parameters is provided [here](../api-reference/sql-jdbc.md#dynamic-parameters).
+
+## Reserved keywords
+
+Druid SQL reserves certain keywords which are used in its query language. Apache Druid inherits all of the reserved keywords from [Apache Calcite](https://calcite.apache.org/docs/reference.html#keywords), in addition to these the following keywords unique to Apache Druid and are also reserved:
+
+* **CLUSTERED**
+* **PARTITIONED**
+
+To use reserved keywords in queries, enclose them in quotation marks, for example the reserved keyword **CYCLE** can be used in the following query only if correct quoted:
```

Comment:
nit: It would be better to use either `CLUSTERED` or `PARTITIONED` in the example.